lib/app/features/home/home_widgets/home_options_tap_bar/home_options_tap_bar.dart

```diff
@@ -6,9 +6,7 @@ import 'package:score_live/core/enums.dart';
 import 'package:score_live/presentation/constants/app_colors.dart';
 
 class HomeOptionsTapBar extends StatelessWidget {
-  HomeOptionsTapBar({super.key});
-
-  final homeCubit = Modular.get<HomeCubit>();
```

Comment:
You could replace `Modular.get` with `context.read`
e.g.
`HomeCubit _homeCubit(BuildContext context) => context.read<HomeCubit>();`

And then in methods `_homeCubit(context).`

Reply:
done